validator-tests/errormessage-hidden-removed.html

```diff
@@ -0,0 +1,40 @@
+<html lang="en-US">
+<head><title>When aria-errormessage is pertinent, authors MUST ensure the content is not hidden so users can navigate to and examine the error message</title></head>
+<body>
+
+<!--
+URL: https://www.w3.org/TR/wai-aria-1.2/#aria-errormessage
+RULE: " Authors MUST use aria-invalid in conjunction with aria-errormessage and when aria-errormessage is pertinent, authors MUST ensure the content is not hidden so users can navigate to and examine the error message"
+-->
+
+<!-- inputs with aria-errormessage that should fail -->
+
+<input type="text" id="error-message-hidden-1" aria-invalid="true" aria-errormessage="id-message-1" class="fail"></div>
+<div id="id-message-1" hidden>Error message 1</div>
+
+<input type="text" id="error-message-hidden-2" aria-invalid="true" aria-errormessage="id-message-2" class="fail"></div>
+<div id="id-message-2" style="display: none">Error message 2</div>
+
+<input type="text" id="error-message-hidden-3" aria-invalid="true" aria-errormessage="id-message-3" class="fail"></div>
+<div id="id-message-3" style="visibility: hidden">Error message 3</div>
+
+<input type="text" id="error-message-empty-1" aria-invalid="true" aria-errormessage="" class="fail"></div>
```

Comment:
How does this fail this specific statement? IMO this is equivalent to aria-errormessage not being present. I don't think this should fail this statement

Reply:
From the spec "Sometimes states and properties are present in the DOM but have a zero-length string ("") as their value. Authors MAY specify a zero-length string ("") for any supported (but not required) state or property. User agents SHOULD treat state and property attributes with a value of "" the same as they treat an absent attribute."